src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -234,13 +246,38 @@ class ExternalIssueActionList extends AsyncComponent {
             <Modal.Header closeButton>
               <Modal.Title>{`${selectedIntegration.provider.name} Issue`}</Modal.Title>
             </Modal.Header>
+            <ul
+              className="nav nav-tabs"
+              style={{'borderBottom': '1px solid rgb(221, 221, 221)'}}
+            >
+              <li className={action == 'create' ? 'active' : ''}>
+                <a id="create" onClick={this.handleClick}>
```

Comment:
I think instead of using `id` attribute, we should just call `handleClick` with the action string.